commit fa2c0772c675b192af032d0bb34457947174dda0 (refs/changes/49/4749/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-08-27T16:42:23+02:00 (1 year, 1 month ago)
    
    TRITON-716 Add bhyve to list of allowed brands to do snapshots

diff --git a/lib/endpoints/vms.js b/lib/endpoints/vms.js
index 1f24c5a..e2cfd35 100644
--- a/lib/endpoints/vms.js
+++ b/lib/endpoints/vms.js
@@ -1472,7 +1472,7 @@ function createVm(req, res, next) {
  */
 function canSnapshot(vm) {
 
-    if (['joyent', 'joyent-minimal', 'lx'].indexOf(vm.brand) === -1) {
+    if (['joyent', 'joyent-minimal', 'lx', 'bhyve'].indexOf(vm.brand) === -1) {
         return (new errors.BrandNotSupportedError(
             'snapshots are not supported for VMs of brand "' + vm.brand + '"'));
     }
